packages/cli-template-contracts-hardhat/tsconfig.json

```
@@ -1,14 +1,12 @@
 {
     "compilerOptions": {
-        "moduleResolution": "Node",
-        "noImplicitAny": true,
-        "resolveJsonModule": true,
-        "target": "ES2018",
-        "module": "CommonJS",
-        "strict": true,
+        "target": "es2020",
+        "module": "commonjs",
         "esModuleInterop": true,
-        "outDir": "dist",
-        "typeRoots": ["node_modules/@types", "types"]
+        "forceConsistentCasingInFileNames": true,
+        "strict": true,
+        "skipLibCheck": true,
+        "resolveJsonModule": true
     },
     "include": ["scripts/**/*", "tasks/**/*", "test/**/*", "build/typechain/**/*", "types/**/*"],
```

```suggestion
    "include": ["scripts/**/*", "tasks/**/*", "test/**/*", "typechain-types/**/*"],
```

The same change needs to be applied to the other templates.